deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -984,6 +980,14 @@ class ChatViewController: UITableViewController, UITableViewDropDelegate {
         self.reloadData()
     }
 
+    private func canReply(message: DcMsg) -> Bool {
```

Comment:
👍 This and `canReplyPrivately` is good!

The only small suggestion (as in: feel free to ignore) I have is to rename the first parameter to `to` like this `canReply(to message: DcMsg)`. The idea is to have a `canReply(to: message)`, to me it's more readable :)

Reply:
sounds good, i will add a commit